chipsec/modules/common/memlock.py

```diff
@@ -51,7 +51,7 @@ def is_supported(self):
         # causes a GP fault and can't be caught in UEFI Shell.
         if self.cs.get_chipset_id() in chipsec.chipset.CHIPSET_FAMILY_ATOM:
             return False
-        return True
+        return False
```

Comment:
Looks like you may have committed an extra file.  This should not be added to this commit.